### PR TITLE
[DINT-425] Added validation to avoid uploading empty parquet files

### PR DESCRIPTION
**Pull request by pedro-contart**
Added an validation clause to avoid creation of empty parquet files on s3.
Empty parquet files can't have it's schema identified by Redshift Spectrum, causing schema validation errors when using the table, even if it's only one partition.

[Related issue](https://quintoandar.atlassian.net/browse/DINT-425)

### CODE REVIEW

**Review by kenjihiraoka — COMMENTED** [review]
lgtm, but this change won't break another pipe that use this client right?

**Comment by pedro-contart** [comment]
> lgtm, but this change won't break another pipe that use this client right?

Shouldn't break anything, since it's only validating on the method that upload's the file
The method doesn't return anything, so there is no response to break, at least i think so

**Comment by kenjihiraoka** [comment]
> > lgtm, but this change won't break another pipe that use this client right?
> 
> Shouldn't break anything, since it's only validating on the method that upload's the file
> The method doesn't return anything, so there is no response to break, at least i think so

Huum yeah, makes sense. If the method don't return anything because he "only" saved to S3 and log some messages, I think it's safe

File: qa_python_utils/aws/athena.py [hunk 1]
```diff
@@ -254,6 +254,15 @@ def create_parquet_from_query(self, key, query, row_group_offsets=500000,
             s3_bucket=s3_bucket,
             bucket_folder_path=bucket_folder_path
         )
+
+        if df.empty:
```
Comment (amommendes) [inline qa_python_utils/aws/athena.py:258]: Do we need to write a test for this? I know this is pretty simple, but what happens if the dataframe is empty? Is this not created if the df is empty? WDYT?

Comment (pedro-contart) [reply qa_python_utils/aws/athena.py:258]: Don't think there is any problem writing a test. Didn't do it because it's an legacy script, and we have a new `athena_client` on bi-etl-ejuice already, so i thought there were no need.

Right now if the df is created, an empty parquet file is created on the partition, what is causing the error (schema error) on Redshift.
This code is to avoid creating this empty file, and keeping the behavior of downloading the csv file and creating the df from it.

So it's up to you guys, if you think a test is needed, i will make one!

Comment (amommendes) [reply qa_python_utils/aws/athena.py:258]: give we have a new client and this will be deprecated soon, totally agree @pedro-contart ... :rocket:

**Review by amommendes — APPROVED** [review]
lgtm :rocket:
